src/orga/governance.rst

```diff
@@ -31,6 +31,11 @@ Here are defined the primary teams participating in conda-forge activities.
   core, part of staged-recipes, or maintainers. This includes first time
   contributors, collaborators, and funders. They have no special rights within
   the conda-forge organization itself.
+* **emeritus-core:** Core members that are inactive (commits, GitHub comments/issues/reviews,
+  and voting on polls) in the past year will be asked if they want to become emeritus-core
```

Comment:
This looks good to me.  Do we want it to be a year? Or is a short time period (6 m.) more approriate?

Reply:
I don't really have a strong opinion here. Maybe 6 months could be safer for our voting process.

Reply:
It seems like folks think 6 months is ok

Reply:
Update to 6 months.